app/controllers/sessions_controller.rb

```diff
@@ -63,7 +63,7 @@ def destroy
   private
 
   def set_home_page_flash_message
-    return unless Settings.home_page_flash_message_html
+    return if Settings.home_page_flash_message_html.blank?
```

Comment:
I think this can be left as it was before, and only the `home_page_flash_message_html` should be removed. This way, in the future if we want to add a different message, we just need to add a new message.

Reply:
I changed it back to how it was.